components/HackathonModal.tsx

```diff
@@ -0,0 +1,94 @@
+// What is this?
+// For our hackathon, we wanted to have a extra way to display and grab the user's attention
+
+// CSS
+import styles from '../styles/components/HackathonModal.module.scss';
+// Config
+import { website_config } from '../config';
+// Components
+import React, { useState, useEffect } from 'react';
+import Modal from 'react-modal';
+import Image from '../components/Image';
+import Link from 'next/link';
+import moment from 'moment';
+// Images
+import LittleGuy from '../public/Hackathon/little-guy.png';
+import TopLeft from '../public/Hackathon/top-left.svg';
+import TopRight from '../public/Hackathon/top-right.svg';
+import BottomLeft from '../public/Hackathon/bottom-left.svg';
+// Fonts - A collection of fonts used throughout the website from Utils
+import { Fonts } from '../utils';
+
+const MODAL_DISPLAY_INTERVAL = moment.duration(2, 'days').asMilliseconds(); // 2 days in milliseconds
```

Comment:
```suggestion
// We only want to show the modal to the user if it has been `2` days since they last saw it.
const MODAL_DISPLAY_INTERVAL = moment.duration(2, 'days').asMilliseconds();
```